test/verify/testsuite-prepare

```diff
@@ -1,87 +1,66 @@
-#!/bin/bash
-
-set -e
-
-SELF=testsuite-prepare
-BASE="$(dirname $0)/.."
-
-vm_install_opts=
-
-# TODO: This really needs to be migrated to python and use testinfra
-TEST_OS=${TEST_OS:-fedora-24}
-
-warning()
-{
-       echo >&2 "$SELF: $@"
-}
+#!/usr/bin/env python
+# This file is part of Cockpit.
+#
+# Copyright (C) 2015 Red Hat, Inc.
+#
+# Cockpit is free software; you can redistribute it and/or modify it
+# under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation; either version 2.1 of the License, or
+# (at your option) any later version.
+#
+# Cockpit is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
 
-silent()
-{
-       "$@" > /dev/null 2> /dev/null
-       return $?
-}
+import argparse
+import os
+import shutil
+import sys
+import imp
 
-usage()
-{
-    echo "usage: testsuite-prepare [--clean] [--quick] [--verbose]"
-}
+basepath = os.path.join(os.path.dirname(os.path.abspath(__file__)),"..")
+if not basepath in sys.path:
+    sys.path.insert(1, basepath)
+from common import testinfra
+vminstall = imp.load_source('vminstall', os.path.join(basepath,"vm-install"))
+vmdownload = imp.load_source('vmdownload', os.path.join(basepath,"vm-download"))
 
+image_list="ipa openshift fedora-stock"
```

Comment:
Yep, I had it in string, because in older commit I called os.system("../vm-download %s " %...) so that I've let it in old form.